docs/why.qmd

```diff
@@ -2,24 +2,224 @@
+or, more typically, on the order of
+[one](https://github.com/ibis-project/ibis/pull/7954) or
+[two](https://github.com/ibis-project/ibis/pull/7303) months for a new
+contributor.
+
+::: {.callout-warning title="Why not the pandas or PySpark APIs?" collapse="true"}
+The pandas API inherently does not scale due to its single-threaded design,
+ordered index, and a lot of API baggage. The creator of pandas (and Ibis!) has
+[talked about the issues with pandas
+publicly](https://wesmckinney.com/blog/apache-arrow-pandas-internals/). While
+there have been projects attempting to scale the pandas API, they always result
+in a dubious support matrix. You can see that with
+[Modin](https://modin.readthedocs.io/en/stable/supported_apis/dataframe_supported.html)
+or [pandas on Spark (formerly known as
+Koalas)](https://spark.apache.org/docs/latest/api/python/user_guide/pandas_on_spark/supported_pandas_api.html).
+
+[Google BigQuery
+DataFrames](https://github.com/googleapis/python-bigquery-dataframes) is a more
+modern attempt to scale the pandas API built on top of Ibis. If you are going
+to build a pandas API we recommend you take a look at this project.
+
+PySpark is a great API for Spark, but not very Pythonic and tightly coupled to
+the Spark execution engine.
+
+Ibis takes inspiration from pandas and PySpark -- and R and SQL -- but is designed to be
+scalable from the start. If offers a neutral,
+[self-governed](https://github.com/ibis-project/governance) open source option
+for your data platform.
+:::
+
+### Ibis for data organizations
+
+Data organizations can use Ibis to standardize the interface for SQL and Python
+data practitioners. It also allows data organizations to:
+
+- transfer data between systems
+- transform, analyze, and prepare data where it lives
+- benchmark your workload(s) across data systems using the same code
+- mix SQL and Python code seamlessly, with all the benefits of a general-purpose
+ programming language, type checking, and expression validation
+
+### Ibis for data library developers
+
+Python developers creating data libraries can use Ibis to:
+
+- instantly support 20+ data backends
+- instantly support pandas, PyArrow, and Polars objects
+- read and write from all common file formats (depending on the backend)
+- trace column-level lineage through Ibis expressions
+- compile Ibis expressions to SQL or Substrait
+- perform cross-dialect SQL transpilation (powered by
+ [SQLGlot](https://github.com/tobymao/sqlglot))
+
+## How does Ibis work?
 
 Most Python dataframes are tightly coupled to their execution engine. And many
 databases only support SQL, with no Python API. Ibis solves this problem by
 providing a common API for data manipulation in Python, and compiling that API
 into the backend's native language. This means you can learn a single API and
 use it across any supported backend (execution engine).
 
-![](./portability.png)
+Ibis supports three types of backend:
+
+1. SQL-generating backends
```

Comment:
these types are very technical, I feel integrating partners care more about these differences than the end users? I personally think if we break down the target audience more, then provide them with very specific targeted information, it'll make some difficult concepts easily understood by the public.

Reply:
I think end users still tend to wonder "how it works" and may call it overly ambitious as a product -- this + the graph/explanation below helps build confidence that it is primarily an API -> SQL translation and we can do that well across backends